discord/partial_emoji.py

```diff
@@ -25,13 +25,25 @@
 
 from __future__ import annotations
 
+import json
 import re
+from pathlib import Path
 from typing import TYPE_CHECKING, Any, TypedDict, TypeVar
 
 from . import utils
 from .asset import Asset, AssetMixin
 from .errors import InvalidArgument
 
+EMOJIS_MAP_PATH = Path(__file__).parent / "emojis.json"
+
+with EMOJIS_MAP_PATH.open("r", encoding="utf-8") as f:
+    data = json.load(f)
```

Comment:
Can we delete this after it's used so it can be garbage collected and removed from memory

Just `del data` when it's done being used